src/App.jsx

```diff
@@ -1,43 +1,34 @@
-import React, { Component } from "react";
+import { useState } from "react";
 import Result from "./Components/Result";
-import './App.css'
+import "./App.css";
 
-class App extends Component {
-  static defaultProps = {
-    randomNum: Math.floor(Math.random() * 100) + 1,
-  };
-
-  constructor(props) {
-    super(props);
-    this.state = { userNum: "" };
+const App = () => {
+  const [userNum, setUserNum] = useState(0);
 
-    this.handleChange = this.handleChange.bind(this);
-  }
+  const handleOnChange = (e) => {
+    const parsedInput = Number(e.target.value);
 
-  handleChange(event) {
-    this.setState({ [event.target.name]: event.target.value });
-  }
+    if (isNaN(parsedInput)) {
```

Comment:
This is a good opportunity to check that the eslint VSCode extension is working. When I try to use === NaN, I get this red underline and warning:
![image](https://github.com/bpb2008/GuessANumber/assets/152167/9fb4f387-15de-4faa-82b4-4b0f2b0615cf)
Is it the same for you?